content/en/blog/2024/otel-operator-q-and-a/index.md

```diff
@@ -0,0 +1,340 @@
+[base-64 encode](https://www.base64encode.org/) the secret.
+
+2-
+[Expose the secret as an environment variable](https://kubernetes.io/docs/concepts/configuration/secret/#using-a-secret)
+by adding it to the `OpenTelemetryCollector` CR’s
+[`env` section](https://github.com/avillela/otel-target-allocator-talk/blob/21e9643e28165e39bd79f3beec7f2b1f989d87e9/src/resources/02-otel-collector-ls.yml#L16-L21).
+For example:
+
+```yaml
+env:
+  - name: LS_TOKEN
+    valueFrom:
+      secretKeyRef:
+        key: LS_TOKEN
+        name: otel-collector-secret
+```
+
+3- Reference the environment variable in your
+[exporter definition](https://github.com/avillela/otel-target-allocator-talk/blob/21e9643e28165e39bd79f3beec7f2b1f989d87e9/src/resources/02-otel-collector-ls.yml#L43-L47):
+
+```yaml
+exporters:
+  otlp/ls:
+    endpoint: 'ingest.lightstep.com:443'
+    headers:
+      'lightstep-access-token': '${LS_TOKEN}'
+```
+
+For more info, check out my full example
+[here](https://github.com/avillela/otel-target-allocator-talk/blob/main/src/resources/02-otel-collector-ls.yml),
+along with full instructions
+[here](https://github.com/avillela/otel-target-allocator-talk/tree/main?tab=readme-ov-file#3b--kubernetes-deployment-servicenow-cloud-observability-backend).
+
+### Q3: Is the Operator version at parity with the Collector version?
+
+The default version of the Collector used by the Operator is typically behind by
+one version at most. For example, at the time of this writing, the latest
+Operator version is 0.98.0, and the latest Collector version is 0.99.0. In
+addition, the default image of the Collector used by the Operator is the
+[core distribution](/blog/2024/otel-collector-anti-patterns/#3--not-using-the-right-collector-distribution-or-not-building-your-own-distribution)
+(as opposed to the contrib distribution).
+
+### Q4: Can I override the base OTel Collector image?
+
+Yes! In fact,
+[you probably should](https://cloud-native.slack.com/archives/C033BJ8BASU/p1713894678225579)!
+
+As we saw earlier, the
+[core distribution](https://github.cm/open-telemetry/open-telemetry-collector)
+is the default Collector distribution used by the `OpenTelemetryCollector` CR.
+The Core distribution is a bare-bones distribution of the Collector for OTel
+developers to develop and test. It contains a base set of components–i.e.
+[extensions](/docs/collector/configuration/#service-extensions),
+[connectors](/docs/collector/configuration/#connectors),
+[receivers](/docs/collector/configuration/#receivers),
+[processors](/docs/collector/configuration/#processors), and
+[exporters](/docs/collector/configuration/#exporters).
+
+If you want access to more components than the ones offered by core, you can use
+the contrib distribution instead. The contrib distribution extends the core
+distribution, and includes components created by third-parties (including
+vendors and individual community members), that are useful to the OpenTelemetry
+community at large.
```

Comment:
I recommend we reference the new [k8s distribution](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-k8s) here instead

Reply:
@swiatekm-sumo In order to avoid confusion, I'd clarified the question to "If I enable `prometheusCR` in the Target Allocator, do I need the `PodMonitor` and `ServiceMonitor` CRs installed in my Kubernetes cluster?" Let me know what you think!